libappstream-glib/as-store.c

```diff
@@ -2822,10 +2822,10 @@ as_store_search_per_user (AsStore *store,
 			return FALSE;
 	}
 
-	/* AppData */
+	/* AppData / MetaInfo */
 	if ((flags & AS_STORE_LOAD_FLAG_APPDATA) > 0) {
 		g_autofree gchar *dest = NULL;
-		dest = g_build_filename (g_get_user_data_dir (), "appdata", NULL);
+		dest = g_build_filename (g_get_user_data_dir (), "metainfo", NULL);
```

Comment:
Jup (I only consider the first patch to be merge-ready... Originally I wanted to update the second patch to have a fallback, but somehow forgot about this over the weekend...)